docs/extensions/publishing-extensions.md

```diff
@@ -0,0 +1,54 @@
+---
+contributors: Esther Agbaje
+description: Discover how to make your Directus extensions available to other users and in the Marketplace.
+---
+
+# Publishing your Extension
```

Comment:
Let's nip this page out of this PR, and then we'll open  another towards the release including secure extensions.